src/main/java/cc/polyfrost/oneconfig/renderer/LwjglManager.java

```diff
@@ -0,0 +1,47 @@
+package cc.polyfrost.oneconfig.renderer;
+
+import cc.polyfrost.oneconfig.renderer.asset.AssetHelper;
+import cc.polyfrost.oneconfig.renderer.font.FontHelper;
+import cc.polyfrost.oneconfig.renderer.scissor.ScissorHelper;
+
+import java.util.ServiceLoader;
+
+/**
+ * Abstraction over the LWJGL3 implementation & loading.
+ *
+ * @deprecated Use the direct interface instances instead.
+ */
+@SuppressWarnings("DeprecatedIsStillUsed"/*, reason = "Methods are still used internally in their respective interfaces" */)
+@Deprecated
```

Comment:
@org.jetbrains.annotations.ApiStatus.Internal ?

Reply:
this may depend on the small fixes PR

Reply:
The ApiStatus annotation should be added by #138, but I'm also gonna refactor it to `cc.polyfrost.oneconfig.internal.renderer` just to be safe